numpy/distutils/mingw32ccompiler.py

```diff
@@ -15,7 +15,6 @@
 import textwrap
 
 # Overwrite certain distutils.ccompiler functions:
-import numpy.distutils.ccompiler  # noqa: F401
```

Comment:
please undo this, as the comment right above it says, this import has a function.